apps/code-infra-dashboard/tsconfig.json

```diff
@@ -15,7 +15,9 @@
     "isolatedModules": true,
     "noEmit": true,
     "jsx": "react-jsx",
-    "downlevelIteration": true
+    "outDir": "./build",
+    "downlevelIteration": true,
```

Comment:
Why do we need `downlevelIteration`?

Reply:
Our target is set to es5, I'm updating in https://github.com/mui/mui-public/pull/334